packages/react-core/src/modules/feed/hooks/useNotifications.ts

```diff
@@ -13,12 +13,13 @@ function useNotifications(
       feedClientRef.current.dispose();
     }
 
-    const feedClient = knock.feeds.initialize(feedChannelId, options);
+    feedClientRef.current = knock.feeds.initialize(feedChannelId, options);
+    feedClientRef.current.listenForUpdates();
+    feedClientRef.current.store.subscribe((t) =>
+      feedClientRef?.current?.store.setState(t),
+    );
```

Comment:
mentioned in slack but wondering if it's worth wrapping this in if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development')`?